java/arcs/core/storage/DirectStoreMuxer.kt

```diff
@@ -111,22 +108,15 @@ class DirectStoreMuxer<Data : CrdtData, Op : CrdtOperationAtTime, T>(
     }
 
     /**
-     * Sends the provided [ProxyMessage] to the store backing the provided [referenceId].
+     * Sends the [ProxyMessage] to the store backing [muxId].
      *
-     * A new store will be created for the [referenceId], if necessary.
+     * A new store will be created for the [muxId], if necessary.
```

Comment:
Just a minor formatting thing:

`muxId` is no longer an argument for this method, nor is it a type, so I believe it should just be code font instead (in backticks) rather than linked (square brackets). 

Or I think you can do `[MuxedProxyMessage.muxId]`??